try{
    //let i = 1/0;
    throw new Error('ada sesuatu yang salah')
    //console.log('berhasil');
    
}catch(err){
    console.log(err);
    
}